src/vs/workbench/contrib/webviewPanel/browser/webviewWorkbenchService.ts

```diff
@@ -226,7 +226,14 @@ export class WebviewEditorService extends Disposable implements IWebviewWorkbenc
 		contentOptions: WebviewContentOptions,
 		extension: WebviewExtensionDescription | undefined,
 	): WebviewInput {
-		const webview = this._webviewService.createWebviewOverlay(id, webviewOptions, contentOptions, extension);
```

Comment:
This should not be required since `webviewOptions` was already being passed along